packages/thirdweb/src/exports/extensions/erc1155.ts

```diff
@@ -174,27 +174,5 @@ export {
   type TokensMintedWithSignatureEventFilters,
 } from "../../extensions/erc1155/__generated__/ISignatureMintERC1155/events/TokensMintedWithSignature.js";
 
-// Packs
-export {
-  packCreatedEvent,
-  type PackCreatedEventFilters,
-} from "../../extensions/erc1155/__generated__/IPack/events/PackCreated.js";
-export {
-  packOpenedEvent,
-  type PackOpenedEventFilters,
-} from "../../extensions/erc1155/__generated__/IPack/events/PackOpened.js";
-export {
-  packUpdatedEvent,
-  type PackUpdatedEventFilters,
-} from "../../extensions/erc1155/__generated__/IPack/events/PackUpdated.js";
-export {
-  createPack,
-  type CreatePackParams,
-} from "../../extensions/erc1155/__generated__/IPack/write/createPack.js";
-export {
-  openPack,
```

Comment:
do we need to remove these? we can just have them exposed in both side right? worried that some ppl might be using these

Reply:
I was aiming for a breaking change but good thing I stacked this PR eh!
Gonna close this one then